vendor-extra/LiberoPageBundle/src/EventListener/SiteHeaderListener.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\EventListener;
+
+use Libero\LiberoPageBundle\Event\CreatePagePartEvent;
+use Libero\ViewsBundle\Views\ContextAwareTranslation;
+use Libero\ViewsBundle\Views\TemplateView;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use const Libero\LiberoPatternsBundle\MAIN_GRID_FULL;
+
+final class SiteHeaderListener
```

Comment:
We will need to revisit this, to probably make it dispatch an event and the details get added that way (eg to allow for custom menus), but good enough for now.